app/helpers/arf_reports_helper.rb

```diff
@@ -52,4 +52,17 @@ def multiple_actions_arf_report_select
          :'data-dialog-title' => _("%s - The following compliance reports are about to be changed") % action[0])
       end.flatten)
   end
+
+  def openscap_proxy_link(arf_report)
+    return _("No proxy found!") unless arf_report.openscap_proxy
+    display_link_if_authorized(arf_report.openscap_proxy.name, hash_for_smart_proxy_path(:id => arf_report.openscap_proxy_id))
+  end
+
+  def reported_info(arf_report)
+    msg = _("Reported at %s") % arf_report.reported_at
+    msg << _(" for policy %s") % display_link_if_authorized(arf_report.policy.name, hash_for_edit_policy_path(:id => arf_report.policy.id))
```

Comment:
also if policy is removed later, it can return nil causing 500 (nil.name), I haven't tested by I encountered it in my DB. The same could happen for deleted proxy.